test/connectivity/modules/NetworkController/docker/linux/arm64v8/Dockerfile

```diff
@@ -1,5 +1,5 @@
-ARG base_tag=1.0.5-linux-arm64v8
-FROM azureiotedge/azureiotedge-module-base:${base_tag}
+ARG base_tag=1.0.5.1-linux-arm64v8
+FROM edgebuilds.azurecr.io/microsoft/azureiotedge-module-base-full:${base_tag}
```

Comment:
WHy this change?

Reply:
We publish images to 2 different places. I talked with Anca and it seemed like the NetworkController pull its base image from a different CR than the rest of the test modules. This fix consolidates that.

Reply:
This image is not used in any place as Connectivity test is configured to run on Linux amd64 and arm32.   Agreed, this should be fixed.